tests/runtime/data/stackdriver/stackdriver_multi_entries_severity.log

```diff
@@ -0,0 +1,3 @@
+{"message":"test severity1", "severity":"INFO"}
```

Comment:
I think this testcase doesn't cover the regression we discussed offline for https://github.com/fluent/fluent-bit/pull/2380 , which I think should be the testcases we would like to add here.

For example, if the 2nd log doesn't have severity key, it would expose the bug we currently have, and it can be fixed by https://github.com/fluent/fluent-bit/pull/2380.

Reply:
updated